app/controllers/public/index.js

```diff
@@ -10,6 +10,10 @@ export default Controller.extend({
   { name: 'Speaker 4', organisation: 'Org 4', isFeatured: true, socialLinks: [{ name: 'twitter', url: '#' }], short_biography: 'Full-stack developer with experience in PHP, Python &amp; Ember.JS. Plays around with Kubernetes and Containers. Avid open source contributor.' },
   { name: 'Speaker 5', organisation: 'Org 5', isFeatured: false, socialLinks: [{ name: 'home', url: '#' }],  short_biography: 'Full-stack developer with experience in PHP, Python &amp; Ember.JS. Plays around with Kubernetes and Containers. Avid open source contributor.' }],
 
+  sponsors: [{ title: 'Gold Sponsors', sponsorList: [{ name: 'Sponsor 1', url: '#', logo: 'http://placehold.it/150x60' }, { name: 'Sponsor 2', url: '#', logo: 'http://placehold.it/150x60' }, { name: 'Sponsor 3', url: '#', logo: 'http://placehold.it/150x60' }] },
```

Comment:
Correct. But you don't have to use a custom function. Use [lodash's `groupBy`](https://lodash.com/docs/4.17.4#groupBy). Lodash is included in the project. Feel free to use all of the awesome utility functions there 😄 .

Since we'll be sorting and grouping, if you're using `computed.sort` we'll probably need two computed properties. (One for sorting and that passed through another which does grouping.)

Instead use one normal computed property and use lodash's `orderBy` and `groupBy`